covid_graphs/covid_graphs/scripts/country_dashboard.py

```diff
@@ -66,9 +66,8 @@ def _get_header_content(prediction_event: PredictionEvent, title: str):
             'On 2020-03-30, mathematicians Katarína Boďová and Richard Kollár ',
             html.A('made predictions about 7 countries', href=prediction_link),
             f'. The data available up to that point (until {prediction_date_str}) is in the ',
-            html.Span('green zone', style={'color': 'green'}),
-            f'. Data coming after {prediction_date_str} is in the ',
-            html.Span('blue zone.', style=dict(color='blue')),
+            html.Span('light green zone', style={'color': 'lightgreen'}),
```

Comment:
Just wondering if `background-color` attribute would be more fitting.

Reply:
Will try it out tomorrow.